5.6/apache/docker-yii2-php.conf

```diff
@@ -0,0 +1,27 @@
+<FilesMatch \.php$>
+        SetHandler application/x-httpd-php
+</FilesMatch>
+
+DirectoryIndex disabled
+DirectoryIndex index.php index.html
```

Comment:
Why is `DirectoryIndex` duplicated here? the disabled line does not make sense to me.